src/applications/vaos/new-appointment/components/DateTimeSelectPage/index.jsx

```diff
@@ -13,18 +13,24 @@ import CalendarWidget from '../../../components/calendar/CalendarWidget';
 import WaitTimeAlert from './WaitTimeAlert';
 import { FETCH_STATUS } from '../../../utils/constants';
 import { getRealFacilityId } from '../../../utils/appointment';
+import useIsInitialLoad from '../../../hooks/useIsInitialLoad';
 
 const pageKey = 'selectDateTime';
 const pageTitle = 'Tell us the date and time you’d like your appointment';
 
 const missingDateError =
   'Please choose your preferred date and time for your appointment.';
 
-function ErrorMessage({ facilityId, requestAppointmentDateChoice }) {
+function ErrorMessage({ facilityId, requestAppointmentDateChoice, history }) {
```

Comment:
History was not passed through, causing an error. ESLint didn't flag it because `history` is a valid global.